bib/cmd/bootc-image-builder/image.go

```diff
@@ -265,8 +261,6 @@ func manifestForISO(c *ManifestConfig, rng *rand.Rand) (*manifest.Manifest, erro
 		},
 	}
 
-	img.ISOLabelTmpl = "Container-Installer-%s"
```

Comment:
I think we want something like `img.ISOLabel = fmt.Sprintf("Container-Installer-%s", c.Architecture.String())` here instead. I'm not deeply familiar with ISOs, but I think we should keep the label.

Reply:
Thank you! I fixed this in https://github.com/osbuild/bootc-image-builder/pull/342